src/sentry/web/frontend/auth_organization_login.py

```diff
@@ -84,8 +84,11 @@ def handle_sso(self, request, organization, auth_provider):
                 auth_provider=auth_provider,
                 flow=AuthHelper.FLOW_LOGIN,
             )
-            helper.init_pipeline()
-            return helper.next_step()
+
+            if request.POST.get('init_login'):
```

Comment:
What if we just call it init instead (less verbose)

Reply:
👍  Updating test cases as well right now to pass along the init stuff.

Reply:
96bb617